test/cli/cli.test.js

```diff
@@ -82,6 +82,12 @@ describe('CLI', function () {
     assert.deepEqual(frEmpty, {})
   })
 
+  it('works with `--get-key-duplicates` option', async () => {
+    const subprocess = await execaCommand('yarn test:cli --get-key-duplicates')
+
+    assert.include(subprocess.stdout, "[ [ 'seventh', 3 ] ]")
```

Comment:
I'm confused why this would be `3` and not `2`. The file has only two occurrences of the key.

Reply:
I went back and looked at how I counted the keys in my own personal implementation compared to the test translation file, and they are slightly different.  That meant that I was adding an extra "count" to each duplicate.  I fixed that in the cli function and it should be sensical now as it correctly counts `2` here etc.